docs/extending/developing-code/extending/case-studies/KroneckerProduct/index.rst

```diff
@@ -0,0 +1,20 @@
+Adding Builtin Function ``KroneckerProduct``
+=============================================
+
+After adding the built-in symbol `Undefined``, we move onto our first built-in function, ``KroneckerProduct``.
+
+Below are the steps in order that were used to create ``KronkeckerProduct``:
+
+
+.. toctree::
+   :maxdepth: 1
+
+   finding
+   where
+   skeletal
+   checking
+   completing
+   testing
+
+
+See also :ref:`Adding Builtin ``Undefined``` for the steps to add the simpler built-in symbol ``Undefined``.
```

Comment:
Seems that there is 1 extra ` after Undefined

Reply:
Should be deleted - thanks.